kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/ImageServiceBase.java

```diff
@@ -41,11 +41,19 @@
 import static org.jbpm.process.svg.processor.SVGProcessor.ACTIVE_BORDER_COLOR;
 import static org.jbpm.process.svg.processor.SVGProcessor.COMPLETED_BORDER_COLOR;
 import static org.jbpm.process.svg.processor.SVGProcessor.COMPLETED_COLOR;
+import static org.kie.server.api.KieServerConstants.KIE_SERVER_IMAGESERVICE_MAX_NODES;
 
 public class ImageServiceBase {
 
     private static final Logger logger = LoggerFactory.getLogger(ImageServiceBase.class);
 
+    /**
+     * This causes the image service to limit the number of nodes (performance). 
+     * Due to this limitation it could cause a known issue not blurring all the nodes active or completed depending on
+     * the process size.
+     */
+    private static final int MAX_NODES = Integer.parseInt(System.getProperty(KIE_SERVER_IMAGESERVICE_MAX_NODES, "10000"));
```

Comment:
I would make the default `1000` as it was before. Just for the sake of backward compatibility.

Reply:
OK, if we take it as a fix, I see no problem. But the behaviour changes slightly now :)

Reply:
I meant I will change it.

Reply:
OK then. Thanks.